sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -115,8 +116,21 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
         MapAttributeParser.parseMapOptions(typedArray, context.resources.displayMetrics.density)
       val cameraOptions = CameraAttributeParser.parseCameraOptions(typedArray)
       val textureView = typedArray.getInt(R.styleable.mapbox_MapView_mapbox_mapSurface, 0) != 0
-      return MapInitOptions(context, resourceOptions, mapOptions, attrs = attrs).also {
-        it.initialCameraOptions = cameraOptions
+      val styleUri =
+        typedArray.getString(R.styleable.mapbox_MapView_mapbox_styleUri) ?: Style.MAPBOX_STREETS
```

Comment:
how can user set the  styleUri to null(match the old behaviour)?

Reply:
User can set this to empty string, we have check in the following line.

Reply:
oh, sorry didn't notice the check below.